monitoring/v3/src/test/java/AlertIT.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import static org.junit.Assert.assertTrue;
+
+import com.google.common.base.Strings;
+import com.google.common.io.Files;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.nio.charset.StandardCharsets;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/**
+ * Tests for monitoring "AlertSample" sample.
+ */
+@RunWith(JUnit4.class)
+@SuppressWarnings("checkstyle:abbreviationaswordinname")
+public class AlertIT {
+  private static String testPolicyName = "test-policy";
+  private static String policyFileName = "target/policyBackup.json";
+  private static Pattern policyNameRegex = Pattern.compile(
+      "alertPolicies/(?<alertid>.*)(?s).*(?!s)notificationChannels/(?<channel>[a-zA-Z0-9]*)$");
+  private ByteArrayOutputStream bout;
+  private PrintStream out;
+
+  @Before
+  public void setUp() {
+    bout = new ByteArrayOutputStream();
+    out = new PrintStream(bout);
+    System.setOut(out);
+  }
+
+  @After
+  public void tearDown() {
+    System.setOut(null);
+  }
+
+  @Test
+  public void testListPolicies() throws IOException {
+    AlertSample.main(new String[]{"list"});
+    assertTrue(bout.toString().contains(testPolicyName));
+  }
+
+  @Test
+  public void testBackupPolicies() throws IOException {
+    AlertSample.main(new String[]{"backup", "-j", policyFileName});
+    File backupFile = new File(policyFileName);
+    assertTrue(backupFile.exists());
+    String fileContents =
+        String.join("\n", Files.readLines(backupFile, StandardCharsets.UTF_8));
+    assertTrue(fileContents.contains("test-policy"));
+  }
+
+  // TODO: Complete restore backup test when parse/unparse issue is figured out.
```

Comment:
If there is a bug associated with this task, it might be useful to update the comment with the id for future use.

Reply:
Done.